tests/Unit/PointwiseFunctions/AnalyticSolutions/WaveEquation/Test_IsotropicSphericalWave.cpp

```diff
@@ -0,0 +1,43 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cstddef>
+#include <tuple>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"
+#include "Evolution/Systems/ScalarWave/Tags.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/WaveEquation/IsotropicSphericalWave.hpp"
+#include "PointwiseFunctions/MathFunctions/Gaussian.hpp"
+#include "PointwiseFunctions/MathFunctions/MathFunction.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+template <size_t Dim>
+void test_solution() {
+  const ScalarWave::Solutions::IsotropicSphericalWave<Dim> solution{
+      std::make_unique<MathFunctions::Gaussian>(1., 1., 0.),
+      std::make_unique<MathFunctions::Gaussian>(-1., 1., 0.)};
+
+  test_creation<ScalarWave::Solutions::IsotropicSphericalWave<Dim>>(
+      "  IngoingProfile:\n    Gaussian:\n      Amplitude: 1.\n      Width: "
+      "1.\n      Center: 0.\n  OutgoingProfile:\n    Gaussian:\n      "
+      "Amplitude: -1.\n      Width: 1.\n      Center: 0.\n");
+}
+
+}  // namespace
+
+SPECTRE_TEST_CASE(
+    "Unit.AnalyticSolutions.WaveEquation.IsotropicSphericalWave",
+    "[PointwiseFunctions][Unit]") {
```

Comment:
This test doesn't test anything except option parsing.

Reply:
The TODO is mentioned in the description